.github/workflows/test.yml

```diff
@@ -56,3 +56,54 @@ jobs:
         echo ---
         read -ra ORAS_VERSION_INSTALLED <<<$(oras version)
         [ "${ORAS_VERSION_INSTALLED[1]}" == "$ORAS_VERSION_EXPECTED" ]
+
+  test-custom-url:
+    name: Test Setup using URL
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup ORAS using URL
+      uses: ./
+      with:
+        url: https://github.com/oras-project/oras/releases/download/v1.0.0/oras_1.0.0_linux_amd64.tar.gz
+        checksum: "8533c9ea1e5a0d5eb1dfc5094c0e8ef106d15462f8a119077548f88937ed2133"
```

Comment:
What if someday we remove 1.0.0? To reduce long-term maintenance effort, it's better to grab an existing version in `release.json`. E.g.

```bash
jq -r 'keys_unsorted[0] as $k | .[$k].linux.amd64' src/lib/data/releases.json
```